tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=py{27,37}-celery{3,4}-redis{2,3}
+envlist=celery{3,4}-redis{2,3}
```

Comment:
With this change, how do I test both python versions locally during development?

Reply:
Setup two python virtualenv?

Or we can use tox to test in different enviroments and use only a single docker image. (change here?  https://github.com/sibson/redbeat/blob/master/.circleci/config.yml )

What do you think? For now on master branch the py37 just skipped:

```
ERROR:   py27-celery3-redis2: commands failed
ERROR:   py27-celery3-redis3: commands failed
  py27-celery4-redis2: commands succeeded
  py27-celery4-redis3: commands succeeded
SKIPPED:  py37-celery3-redis2: InterpreterNotFound: python3.7
SKIPPED:  py37-celery3-redis3: InterpreterNotFound: python3.7
SKIPPED:  py37-celery4-redis2: InterpreterNotFound: python3.7
SKIPPED:  py37-celery4-redis3: InterpreterNotFound: python3.7
```

Reply:
Hi I reverted the changes, and let circleci to test in different python versions. But I changed docker image from python3.6 to python3.7 since your tox env set to py37.

So it should be like this:
- On python3.7 image tox will skip py27 only test py37
- On python2.7 image tox will skip py37 only test py27

---
Now I get you, before I thought the version in tox.ini file was a mistake. Sorry.